resident-services/README.md

```diff
@@ -121,4 +136,21 @@
 
 * Masterdata Getting Templates (Execution) - This API is to return terms and conditions and will be invoked by UI for the specific language. This is a Proxy API of master data service for getting templates for template type code and language code. Template type code will be passed in this API from a file named as template_type and the variable used is "templateTypeCode". The id and access token will be used here and can be used multiple times untill its valid and not expired.
 
-* Share Credential (Execution) - This API is to share the user specified attributes to selected partner in the selected formats along with a purpose. The id and access token will be used here and can be used multiple times untill its valid and not expired.
\ No newline at end of file
+* Share Credential (Execution) - This API is to share the user specified attributes to selected partner in the selected formats along with a purpose. The id and access token will be used here and can be used multiple times untill its valid and not expired.
+
+* Login (Execution) - Using this API, the resident portal will redirect to the Esignet URL where the validation is performed and login happens.
+
+* Menu Bar Unread Notification Count (Execution) - API to return the number of unread service notification list. The id and access token will be used here and can be used multiple times untill its valid and not expired.
+
+* Menu Bar Bell Notification Click (Execution) - This returns the last date time of user click event on the bell notification icon. The id and access token will be used here and can be used multiple times untill its valid and not expired.
+
+* Menu Bar Bell Update Time (Execution) - API to update last date-time when user clicked on the bell notification. The id and access token will be used here and can be used multiple times untill its valid and not expired.
```

Comment:
this is the current time when user clicked or the last time ?

Reply:
Whenever user clicked on the bell notification that date-time.

Reply:
can we change the description accordingly ?

Reply:
Done

Reply:
Lets update as latest

Reply:
Done